src/model/immutable/BlockMapBuilder.js

```diff
@@ -22,7 +22,7 @@ var {OrderedMap} = Immutable;
 var BlockMapBuilder = {
   createFromArray: function(
     blocks: Array<ContentBlock>
-  ): BlockMap {
+  ): BlockMap | OrderedMap {
```

Comment:
We should check why we can return those 2 different types, and add a comment explaining.